contrib/genpkglist.py

```diff
@@ -71,36 +71,24 @@ def exclude(self, devices: [str], pkglists: [PackageList]=None):
         return self
 
     def render(self):
-        if bool(self.includes):
```

Comment:
Why has this been removed? Without this we can include and exclude a package list at the same time which shouldn't be possible

Reply:
Yes, it was removed intentionally to allow to include and exclude at the same time.

Reply:
The alternative would have been, for the targets where the new include option was used, to switch to exclude again and list ALL the other (not included) devices in the genpkglist file.

Reply:
Sounds fair